snuba/utils/metrics/wrapper.py

```diff
@@ -28,7 +28,9 @@ def __merge_tags(self, tags: Optional[Tags]) -> Optional[Tags]:
         elif tags is None:
             return self.__tags
         else:
-            return ChainMap(tags, self.__tags)
+            # TODO: ChainMap takes a MutableMapping, but both snuba and arroyo type tags as a Mapping.
+            # Fixing this requires updating both packages.
```

Comment:
Is this actually an issue in snuba/arroyo or is it a mypy bug/limitation?

Reply:
It's not a mypy bug, `ChainMap` is a core library that is typed to `MutableMapping`. I think the stubs got updated or something. So it is an "issue" with how the `Tags` are typed in snuba/arroyo, but it's just a minor typing issue.

Reply:
Ok